tests/auth_tests/test_management.py

```diff
@@ -968,6 +968,34 @@ def test_blank_username_non_interactive(self):
                 stderr=new_io,
             )
 
+    def test_blank_email_non_interactive(self):
+        new_io = StringIO()
+
+        call_command(
+            "createsuperuser",
+            email="",
+            username="joe",
+            interactive=False,
+            stdin=MockTTY(),
+            stdout=new_io,
+            stderr=new_io,
+        )
+        self.assertEqual(new_io.getvalue().strip(), "Superuser created successfully.")
```

Comment:
I think these two tests would need a few extra assertions ensuring the superuser was actually created (besides the existing assert over the returned success message. See `test_basic_usage` as example of the assertions we could use.